app/controllers/api/secret/SecretController.js

```diff
@@ -0,0 +1,303 @@
+'use strict';
+ var openpgp = require('openpgp');
+ var pg  = require('pg');
+ var https = require('https');
+ var url = require('url');
+ var authSRV = 'https://nanopeppa.freefeed.net/v1/posts/';
```

Comment:
Yup, that's true

On Thu, Jul 30, 2015 at 5:56 PM, Alexey Zakhlestin <notifications@github.com
> wrote:

> In app/controllers/api/secret/SecretController.js
> <https://github.com/pepyatka/pepyatka-server/pull/277#discussion_r35879360>
> :
>
> > @@ -0,0 +1,303 @@
> > +'use strict';
> > + var openpgp = require('openpgp');
> > + var pg  = require('pg');
> > + var https = require('https');
> > + var url = require('url');
> > + var authSRV = 'https://nanopeppa.freefeed.net/v1/posts/';
>
> I believe this line is not needed anymore? just as https require
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pepyatka/pepyatka-server/pull/277/files#r35879360>.
>